### ISSUE TITLE
SBML not valid; subsys_PWY_6151

### ISSUE THREAD

**Issue by thh32** [jotech/gapseq#114]
Hi,

I have been using Gapseq with R and it works great, but recently wanted to ensure the quality of the models made using MEMOTE. 

When I try running MEMOTE on models made with Gapseq I get the following errors;
```
memote report snapshot --filename "test.html" Acetatifactor_muris_DSM23669.comb.xml 
critical: The model could not be loaded due to the following SBML errors.
error: Something went wrong reading the SBML model. Most likely the SBML model is not valid. Please check that your model is valid using the `cobra.io.sbml.validate_sbml_model` function or via the online validator at http://sbml.org/validator .
error: 	`(model, errors) = validate_sbml_model(filename)`
error: If the model is valid and cannot be read please open an issue at https://github.com/opencobra/cobrapy/issues .
error: Line 93722, Column 70 - #1010: Duplicate XML attribute.
error:  - Category: XML content, Severity: 2
```

The error is slightly different each time, but generally the same.

When I check the file with the online validator I get the following message;

```
Error[ Line 93722](https://sbml.org/validator_servlet/section-print-result.jsp#line-93722) Column 72: (XML Error #1010) Duplicate XML attribute.
<groups:group sboTerm="SBO:0000411" groups:id="subsys_PWY_6151" groups:id="subsys_PWY_6151" groups:name="PWY-6151" groups:kind="partonomy">
```
Searching the file I find two separate groups names the same, confirming the presence of a duplicate attribute.

Not only does this grouping appear twice in the file, the issue is that the "groups:id" is given twice in this line.

I dont know if this is something you have come across before?

Attached is one of the files. 

[Acetatifactor_muris_DSM23669.comb.zip](https://github.com/jotech/gapseq/files/8396892/Acetatifactor_muris_DSM23669.comb.zip)

**Comment by Waschina** [comment]
Hi @thh32 

thanks for reporting - this is clearly a small bug. I'll try to fix this by the end of next week.

**Comment by Waschina** [comment]
@thh32 Could you maybe also share the RDS model file? That will help to quickly solve the issue.

**Comment by thh32** [comment]
Sur
[Acetatifactor_muris_DSM23669.comb-draft.zip](https://github.com/jotech/gapseq/files/8397616/Acetatifactor_muris_DSM23669.comb-draft.zip)

**Comment by Waschina** [comment]
Hi @thh32 

the issue should already be fixed with commit d5688063a7001559fda22f04561a2a10372cfc7d 

Could you try to update gapseq? This should fix the error. If not, please reopen this issue.
